pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -196,6 +196,14 @@ message LinuxSandboxSecurityContext {
     bool privileged = 6;
 }
 
+// Sysctls holds linux sysctls config for the sandbox.
+message Sysctls {
+    // List of safe sysctls which are set for the sandbox.
+    map<string, string> sysctls = 1;
+    // List of unsafe sysctls which are set for the sandbox.
+    map<string, string> unsafe_sysctls = 2;
```

Comment:
Why have a separate field for unsafe_sysctls? IIUC, the distinction is just so that the Kubelet can perform extra validation. Since it's already been validated by the time it's passed to the CRI, shouldn't we just take a single list of sysctls to set here?